Benkimeric/MnE-Frontend
# Add SideBar and Header Skeleton

#### What does this PR do?
Adds sidebar and header skeleton

#### Description of Task to be completed?
Add skeleton for sidebar and header UI

#### How should this be manually tested?
- Clone this repo `git clone  [the-repo-link]`
- Checkout to this branch `ft-add-sidebar-and-header`
- Install all dependencies `yarn install`
- Run  `yarn start`
- Notice the presence of sidebar and header skeleton UI

#### Any background context you want to provide?

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/19903559/77291635-ba7d9f80-6cef-11ea-81f9-a303deedc062.png)

#### Questions:
Any suggestions on what to improve?
